tensorzero-core/tests/e2e/feedback.rs

```diff
@@ -52,8 +52,16 @@ async fn e2e_test_comment_feedback_with_payload(inference_payload: serde_json::V
     // // Running without valid episode_id. Should fail.
     let episode_id = Uuid::now_v7();
     // Test comment feedback on episode
-    let tag_value = Uuid::now_v7().to_string();
-    let payload = json!({"episode_id": episode_id, "metric_name": "comment", "value": "good job!", "tags": {"key": tag_value}});
+    let payload = serde_json::to_value(Params {
```

Comment:
one nit here: since `Params` implements `Serialize` you don't need to manually do so. You should be able to remove all the instances of `serde_json::to_value(..).unwrap()` wrapping Params and the client will still take them in `.json(&payload)`